zerver/data_import/import_util.py

```diff
@@ -825,3 +827,21 @@ def long_term_idle_helper(
             user_profile_row["last_active_message_id"] = 1
 
     return long_term_idle
+
+
+def check_converted_users_email(user_emails: list[str]) -> None:
+    invalid_emails = []
+    for email in user_emails:
+        try:
+            validate_email(email)
+        except ValidationError:
+            invalid_emails.append(f"- {email}\n")
+
+    if invalid_emails:
+        details = "".join(invalid_emails)
+        error_log = (
+            f"Invalid email format, please fix the following email(s) and try again:\n\n{details}"
+        )
+        # ValidationError isn't used here because it's error message
+        # is formatted as literal string, so it won't output newlines(\n).
```

Comment:
Could we just print a string with the invalid emails separated by commas? I do think getting all the invalid emails at once is a good choice, but I don't expect there to be an extremely high number of them most of the time.

`Invalid email format, please fix the following email(s) and try again: user1@email.c, user4@email`

Reply:
Yes, that works too. I'll switch back to raising `ValidationError` too because we don't use newlines. I've updated the error log in the description